{{cookiecutter.project_slug}}/Makefile

```diff
@@ -116,10 +116,12 @@ clean: clean-code clean-data clean-notebooks
 
 ##### JOBS #####
 
+EXPERIMENT?=0
```

Comment:
Wouldn't it be more useful to name the default train project for example `base`, i.e. set `EXPERIMENT=base`? Meaningful names help to figure out what each job does if you start a lot of them. For me, it's not clear that `$(TRAINING_JOB)-0` means.

Maybe, it has sense to add this postfix to other jobs which require `gpu` resources (e.g. `jupyter`).

It also would be a good idea to describe this functional and how to modify your own commands in `README` file of an inited project.

Reply:
Sure, thank you!

Also, thanks for pointing out on the documentation, I totally forgot about it...

Reply:
cool idea, thanks!

Reply:
fixed